toml/prognostic_edmfx_bomex_explicit.toml

```diff
@@ -0,0 +1,62 @@
+[C_E]
+value = 0.0176
+
+[EDMF_surface_area]
+value = 0.1
+
+[EDMF_min_area]
+value = 1.0e-5
```

Comment:
I used min_area = 1e-10 but don't think CA is clipping area anyway (?)

Reply:
Yeah, the min_area here is different from the one in TC. This is used to regulate divide_by_rhoa. It shouldn't matter.